posthog/hogql/database/schema/web_analytics_preaggregated.py

```diff
@@ -7,6 +7,78 @@
     FieldOrTable,
 )
 
+DEVICE_BROWSER_FIELDS = {
+    "browser": StringDatabaseField(name="browser", nullable=True),
+    "browser_version": StringDatabaseField(name="browser_version", nullable=True),
+    "os": StringDatabaseField(name="os", nullable=True),
+    "os_version": StringDatabaseField(name="os_version", nullable=True),
+    "viewport_width": IntegerDatabaseField(name="viewport_width", nullable=True),
+    "viewport_height": IntegerDatabaseField(name="viewport_height", nullable=True),
+}
+
+GEOIP_FIELDS = {
+    "country_code": StringDatabaseField(name="country_code", nullable=True),
+    "country_name": StringDatabaseField(name="country_name", nullable=True),
+    "city_name": StringDatabaseField(name="city_name", nullable=True),
+    "region_code": StringDatabaseField(name="region_code", nullable=True),
+    "region_name": StringDatabaseField(name="region_name", nullable=True),
+    "time_zone": StringDatabaseField(name="time_zone", nullable=True),
+}
+
+UTM_FIELDS = {
+    "utm_source": StringDatabaseField(name="utm_source", nullable=True),
+    "utm_medium": StringDatabaseField(name="utm_medium", nullable=True),
+    "utm_campaign": StringDatabaseField(name="utm_campaign", nullable=True),
+    "utm_term": StringDatabaseField(name="utm_term", nullable=True),
+    "utm_content": StringDatabaseField(name="utm_content", nullable=True),
+    "referring_domain": StringDatabaseField(name="referring_domain", nullable=True),
+}
+
+ATTRIBUTION_TRACKING_FIELDS = {
+    "gclid": StringDatabaseField(name="gclid", nullable=True),
+    "gad_source": StringDatabaseField(name="gad_source", nullable=True),
+    "gclsrc": StringDatabaseField(name="gclsrc", nullable=True),
+    "dclid": StringDatabaseField(name="dclid", nullable=True),
+    "gbraid": StringDatabaseField(name="gbraid", nullable=True),
+    "wbraid": StringDatabaseField(name="wbraid", nullable=True),
+    "fbclid": StringDatabaseField(name="fbclid", nullable=True),
+    "msclkid": StringDatabaseField(name="msclkid", nullable=True),
+    "twclid": StringDatabaseField(name="twclid", nullable=True),
+    "li_fat_id": StringDatabaseField(name="li_fat_id", nullable=True),
+    "mc_cid": StringDatabaseField(name="mc_cid", nullable=True),
+    "igshid": StringDatabaseField(name="igshid", nullable=True),
+    "ttclid": StringDatabaseField(name="ttclid", nullable=True),
+    "epik": StringDatabaseField(name="epik", nullable=True),
+    "qclid": StringDatabaseField(name="qclid", nullable=True),
+    "sccid": StringDatabaseField(name="sccid", nullable=True),
+    "_kx": StringDatabaseField(name="_kx", nullable=True),
+    "irclid": StringDatabaseField(name="irclid", nullable=True),
+}
```

Comment:
I really wanna get these alphabetically sorted everywhere 🤣

Reply:
Oh, we can do it here, I think the ones on the table insert/creation can't because of the field orders, but I will do it for you :D

Reply:
Yeah, I once tried changing it on those tables too, but it was tricky to regenerate all tables lol